shell/platform/android/io/flutter/view/ResourceExtractor.java

```diff
@@ -409,6 +409,7 @@ private String getAPKPath() {
         }
     }
 
+    @SuppressWarnings("deprecation")
     private static String[] getSupportedAbis() {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP && Build.SUPPORTED_ABIS.length > 0) {
```

Comment:
Does the deprecation warning still happen if we take out the fallback for empty `SUPPORTED_ABIS` on newer API versions? I don't think we want this fallback anyway. From a second cursory look at [`Build.java`](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/os/Build.java) these deprecated APIs just are a subset of `SUPPORTED_ABIS` anyway.

```suggestion
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Reply:
Removed the `SUPPORTED_ABIS.length` check.  The `SuppressWarnings` annotation is still needed for the warning emitted by javac.